### ISSUE TITLE
using database backend storage

### ISSUE THREAD

**Issue by b4cktr4ck5r3** [tiredofit/docker-lemonldap#17]
Hi, after reading the official documentation i saw that if i wanted to use a MongoDB session storage i would need Apache::Session::MongoDB. But the actual container is running on nginx.

Do you have other way to do a database backend storage with the container ?

Thanks.

**Comment by tiredofit** [comment]
Image has MariaDB, Postgres, Redis and MongoDB client / C libraries installed so you can do that without a rebuild of the image..

**Comment by b4cktr4ck5r3** [comment]
so in configuration lemonldap-ng.ini i just need to use:

[configuration]
host = [IP]:[PORT] => IP of external database

to work with MongoDB backend database ?

thanks

**Comment by b4cktr4ck5r3** [comment]
And i have another issue on my manager when i use the docker-compose, is that normal the preview display a 404 Not Found on this ?

![image](https://user-images.githubusercontent.com/59377281/111921297-2a09a480-8a94-11eb-9551-b717e3167076.png)


Thanks for answer

**Comment by tiredofit** [comment]
Yes you would want to change your `SETUP_TYPE=MANUAL` as an environment varialbe and then you would set your lemonldap-ng.ini file as per the LLNG instructions, depending on what you are doing.

The 404 error there is a bug that I haven't gotten around to solving - This is simply a replica of the LLNG Documentation yet stored locally. It broke a couple builds back and I've not had time to solve. The image works great, I serve thousands of users hourly with it.
